ui/desktop/app/routes/scopes/scope/projects/targets/target.js

```diff
@@ -167,16 +184,14 @@ export default class ScopesScopeProjectsTargetsTargetRoute extends Route {
         session
       );
     } catch (e) {
+      /* eslint-disable-next-line ember/no-controller-access-in-routes */
```

Comment:
As per [previous comment](https://github.com/hashicorp/boundary-ui/pull/1960/files#r1378174976) if we end up using `resetController`, what do you guys think of here not touching the controller but calling `resetController`?

Within `resetController`, [as per documentation](https://guides.emberjs.com/v4.12.0/routing/query-params/), it receives the param `isExiting`, so within `resetController`, we can deal with all the logic knowing if we are exiting or not, which I think, covers the usages of reseting controller we are doing within this route.

Thoughts? @ZedLi @cameronperera